matrix/count_negative_numbers_in_sorted_matrix.py

```diff
@@ -0,0 +1,164 @@
+"""
+Given an `m x n` matrix grid which is sorted in decreasing order
+both row-wise and column-wise, return the number of negative numbers in grid.
```

Comment:
Now I understand better...
```suggestion
Given an matrix of numbers in which all rows and all columns are sorted in decreasing
order, return the number of negative numbers in grid.
```